packages/use-contractkit/src/constants.tsx

```diff
@@ -69,6 +72,7 @@ export const WalletIds = {
   CeloWallet:
     '36d854b702817e228d5c853c528d7bdb46f4bb041d255f67b82eb47111e5676b',
   CeloDance: 'TODO',
+  NodeWallet: 'TODO',
```

Comment:
While this isn't pretty. Could you add your app to the walletconenct registry (https://cloud.walletconnect.com/app/registry) and notify us when it's done? IF you don't want NodeWallet to be yet on the registry, could you then change `TODO` to `TODO:NodeWalletId` ?

Reply:
> Could you add your app to the walletconnect registry (https://cloud.walletconnect.com/app/registry) and notify us when it's done? If you don't want NodeWallet to be yet on the registry, could you then change `TODO` to `TODO:NodeWalletId` ?

App is pending approval.  Will update here once it is approved and added to the registry.